renderdoc/driver/gl/wrappers/gl_draw_funcs.cpp

```diff
@@ -709,10 +709,47 @@ bool WrappedOpenGL::Serialise_glDrawArrays(GLenum mode, GLint first, GLsizei cou
   return true;
 }
 
+void WrappedOpenGL::CopyClientMemoryArrays(GLint first, GLsizei count)
+{
```

Comment:
In order to avoid the copy on idle frames, I could modify this so that instead of overriding the vertex attrib pointer for all subsequent frames, we modify it at the last minute during capture. This would mean querying & modifying the eGL_VERTEX_ATTRIB_ARRAY_POINTER and eGL_VERTEX_ATTRIB_ARRAY_BUFFER_BINDING bindings, and restoring them after the draw call.

Shall I do that?